projects/packages/jetpack-mu-wpcom/src/features/starter-page-templates/class-starter-page-templates.php

```diff
@@ -0,0 +1,448 @@
+<?php
+/**
+ * Starter page templates file.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+namespace Automattic\Jetpack\Jetpack_Mu_Wpcom;
+
+/**
+ * Class Starter_Page_Templates
+ */
+class Starter_Page_Templates {
+
+	/**
+	 * Class instance.
+	 *
+	 * @var Starter_Page_Templates
+	 */
+	private static $instance = null;
+
+	/**
+	 * Cache key for templates array.
+	 * Please access with the  ->get_templates_cache_key() getter.
+	 *
+	 * @var string
+	 */
+	public $templates_cache_key = 'starter_page_templates';
+
+	/**
+	 * Starter_Page_Templates constructor.
+	 */
+	private function __construct() {
+		add_action( 'init', array( $this, 'register_scripts' ) );
+		// Register post metas for Launchpad newsletter task and template tracking
+		add_action( 'init', array( $this, 'register_meta_field' ) );
+		// Enqueue scripts and pass templates in a global JS variable
+		add_action( 'enqueue_block_editor_assets', array( $this, 'enqueue_assets' ) );
+		// Sideload images to add them to the Media Library for Gallery block images to work
+		add_action( 'rest_api_init', array( $this, 'register_rest_api' ) );
+		// Clean caches
+		add_action( 'delete_attachment', array( $this, 'clear_sideloaded_image_cache' ) );
+		add_action( 'switch_theme', array( $this, 'clear_templates_cache' ) );
+		// Handle styles for classic themes
+		add_action( 'block_editor_settings_all', array( $this, 'add_default_editor_styles_for_classic_themes' ), 10, 2 );
+	}
+
+	/**
+	 * Gets the cache key for templates array.
+	 *
+	 * @param string $locale The templates locale.
+	 *
+	 * @return string
+	 */
+	public function get_templates_cache_key( string $locale ) {
+		return $this->templates_cache_key . '_' . $locale;
+	}
+
+	/**
+	 * Creates instance.
+	 *
+	 * @return \Automattic\Jetpack\Jetpack_Mu_Wpcom\Starter_Page_Templates
+	 */
+	public static function get_instance() {
+		if ( null === self::$instance ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Register block editor scripts.
+	 */
+	public function register_scripts() {
+		$script_path = '../../build/starter-page-templates/starter-page-templates.js';
```

Comment:
Try this to resolve the failing test? 🤔 

```suggestion
		$script_path = include Jetpack_Mu_Wpcom::BASE_DIR . 'build/starter-page-templates/starter-page-templates.js';
```

Reply:
Thanks. I have tried that in https://github.com/Automattic/jetpack/pull/38475/commits/2a3243606fcfad882e1a6dd056d213175c29422f but it didn't work. I still see the error:

```
FunctionsTest::test_wpcomsh_get_atomic_client_id_defined
  PHPUnit\Framework\Exception: PHP Warning:  filemtime(): stat failed for /tmp/wordpress-latest/src/wp-content/plugins/wpcomsh/vendor/automattic/jetpack-mu-wpcom/src/build/starter-page-templates/starter-page-templates.js in /tmp/wordpress-latest/src/wp-content/plugins/wpcomsh/vendor/automattic/jetpack-mu-wpcom/src/features/starter-page-templates/class-starter-page-templates.php on line 104
```